packages/contracts-bedrock/test/fixtures/test-deploy-auth-system-in.toml

```diff
@@ -0,0 +1,11 @@
+[safe]
```

Comment:
I expect this name to change, but would like to punt on the naming and structure of this config file until more of the ownership config is being added.